src/send/SendConfirmation.tsx

```diff
@@ -360,6 +408,11 @@ const styles = StyleSheet.create({
     paddingVertical: 8,
     ...fontStyles.largeNumber,
   },
+  amountSubscript: {
+    ...fontStyles.regular,
```

Comment:
this is one of the deprecated font styles right? should we use one of the new font styles?